README.md

```diff
@@ -46,30 +46,122 @@ Simply run the following:
+    ```
+    ssh-keygen -t rsa -C "<your-email>"
+    ```
+
+- make sure you are running terraform 11.x version
+    ```
+    terraform version
+    ```
+    If not try to use [tfswitch](https://warrensbox.github.io/terraform-switcher/) for switching between versions easily:
+    `brew install warrensbox/tap/tfswitch`
+
+
+- Make sure you have Helm version greater/equal than 2.16.4
+    `helm version`
+
+
+### Running
 Before running the server the first time you must set up the DB with:
 
-```
+```bash
 $ cloud schema migrate
 ```
 
 Run the server with:
 
-```
+```bash
 $ cloud server --state-store=<your-s3-bucket>
 ```
+tip: if you want to debug, enable `--dev` flag
+
+
+In a different terminal/window, to create a cluster:
+```bash
+cloud cluster create --zones <availabiity-zone> --size SizeAlef500
+i.e.
+cloud cluster create --zones us-east-1c --size SizeAlef500
+```
+You will get a response like this one:
+```bash
+[
+    {
+        "ID": "tetw1yt3yinjdbhctsstcrybch",
+        "Provider": "aws",
+        "Provisioner": "kops",
+        "ProviderMetadata": "eyJab25lcyI6WyJ1cy1lYXN0LTFjIl19",
+        "ProvisionerMetadata": "eyJOYW1lIjoidGV0dzF5dDN5aW5qZGJoY3Rzc3RjcnliY2gta29wcy5rOHMubG9jYWwiLCJWZXJzaW9uIjoibGF0ZXN0IiwiQU1JIjoiIn0=",
+        "AllowInstallations": true,
+        "Version": "1.16.9",
+        "Size": "SizeAlef500",
+        "State": "creation-requested",
+        "CreateAt": 1589188584607,
+        "DeleteAt": 0,
+        "LockAcquiredBy": "4z4f3xf6sfgnbrxfze94zrbb5y",
+        "LockAcquiredAt": 1589211779835,
+        "UtilityMetadata": "eyJkZXNpcmVkVmVyc2lvbnMiOnsiUHJvbWV0aGV1cyI6IjEwLjQuMCIsIk5naW54IjoiMS4zMC4wIiwiRmx1ZW50Yml0IjoiMi44LjciLCJDZXJ0TWFuYWdlciI6InYwLjEzLjEiLCJQdWJsaWNOZ2lueCI6IjEuMzAuMCJ9LCJhY3R1YWxWZXJzaW9ucyI6eyJQcm9tZXRoZXVzIjoiMTAuNC4wIiwiTmdpbngiOiIxLjMwLjAiLCJGbHVlbnRiaXQiOiIyLjguNyIsIkNlcnRNYW5hZ2VyIjoidjAuMTMuMSIsIlB1YmxpY05naW54IjoiMS4zMC4wIn19"
+    }
+]
+```
+Check its creation progress on the first window where the API runs or run `cloud cluster list`
+ to check cluster status
+
+If something breaks and reprovisioning is needed, run
+```bash
+cloud cluster provision --cluster <cluster-ID>
+i.e.
+cloud cluster provision --cluster tetw1yt3yinjdbhctsstcrybch
+```
+
+Then, when the `state` will be `stable`, export the kubeconfig:
+```bash
+kops export kubecfg <cluster-ID>-kops.k8s.local --state s3://<your-s3-bucket>
+i.e.
+kops export kubecfg tetw1yt3yinjdbhctsstcrybch-kops.k8s.local --state s3://angelos-kops-state
+```
+
+Now check if you can get the pods from the cluster with: `kubectl get pods -A`
+
+#### Installation
+To create an installation, run:
+```bash
+cloud installation create --owner <your-name> --dns <your-dns-record> --size 100users --affinity multitenant
+i.e.
+cloud installation create --owner stelios --dns stelios.test.mattermost.cloud --size 100users --affinity multitenant
+```
 
 ### Testing
 
 Run the go tests to test:
 
-```
+```bash
 $ go test ./...
 ```
 
+### Deleting a cluster
```

Comment:
You may want to call out that you have to delete all installations running on a cluster before the cluster can be deleted.

Reply:
Sorry Gabe I didn't know that one! I will document it